src/app/preferences/views/SSLKeys/SSLKeyList/SSLKeyList.test.tsx

```diff
@@ -1,96 +1,77 @@
-import { MemoryRouter } from "react-router";
-
 import SSLKeyList, { Label as SSLKeyListLabels } from "./SSLKeyList";
 
-import type { RootState } from "@/app/store/root/types";
 import * as factory from "@/testing/factories";
+import { sslKeyResolvers } from "@/testing/resolvers/sslKeys";
 import {
   screen,
-  renderWithMockStore,
-  renderWithBrowserRouter,
+  setupMockServer,
+  mockIsPending,
+  waitForLoading,
+  renderWithProviders,
 } from "@/testing/utils";
 
-describe("SSLKeyList", () => {
-  let state: RootState;
+const mockKeys = {
+  items: [
+    factory.sslKey({
+      id: 1,
+      key: "ssh-rsa aabb",
+    }),
+    factory.sslKey({
+      id: 2,
+      key: "ssh-rsa ccdd",
+    }),
+    factory.sslKey({
+      id: 3,
+      key: "ssh-rsa eeff",
+    }),
+    factory.sslKey({
+      id: 4,
+      key: "ssh-rsa gghh",
+    }),
+    factory.sslKey({
+      id: 5,
+      key: "ssh-rsa gghh",
```

Comment:
*technically* these are beginnings of SSH keys, but they were there before anyway so I'll let it slide (: